indico/modules/events/registration/notifications.py

```diff
@@ -49,7 +49,7 @@ def _notify_registration(registration, template_name, to_managers=False, attach_
         registration.email if not to_managers else registration.registration_form.manager_notification_recipients
     )
     from_address = registration.registration_form.notification_sender_address if not to_managers else None
-    with registration.event.force_event_locale():
+    with registration.event.force_event_locale(user=registration.user):
```

Comment:
You need to take `to_managers` into account here - when emailing managers you want to use the event language, not the registrant's language.

Reply:
Hi @ThiefMaster, I made the passing of the user kind-off mandatory to the `force_event_locale` if the user isn't passed fallback to the event default, if there's no event default, fallback to 'en' as specified in `force_locale`.

This would help in case of 
`registration.event.force_event_locale(user=registration.user if not to_managers else None):`

We will have to change all the callers, which leads to another question,

If a manager opens a custom Email dialog box, we would open the dialog box with the template translated in the logged-in user context (manager). Then, the manager will customize the template, and we will send an email with the recipient users' locale. But, we might not have a translation for all the customizations.